specification/azurestack/resource-manager/Microsoft.AzureStack/stable/2017-06-01/Registration.json

```diff
@@ -171,6 +171,46 @@
                         }
                     }
                 }
+            },
+			"patch": {
+                "x-ms-examples": {
+                    "Patch an Azure Stack registration.": {
+                        "$ref": "examples/Registration/Patch.json"
+                    }
+                },
+                "tags": [
+                    "Registrations"
+                ],
+                "description": "Patch an Azure Stack registration.",
+                "operationId": "Registrations_Update",
+                "parameters": [
+                    {
+                        "$ref": "AzureStack.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "AzureStack.json#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "AzureStack.json#/parameters/RegistrationNameParameter"
+                    },
+                    {
+                        "$ref": "AzureStack.json#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/NewRegistrationParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/Registration"
+                        }
+                    },
+					"404": {
```

Comment:
@AndyTao0402 
Please remove all the error cases and add a "default" response which captures all error cases. The schema of the default response must comply with ARM error contract. sample can be found in Batch RP swagger.